Documentation/design-docs/multi-reg-call-nodes.md

```diff
@@ -0,0 +1,211 @@
+Support for multiple destination regs, GT_CALL and GT_RETURN nodes that return a value in multiple registers:
+============================================================================================================
```

Comment:
This document is confusing in that it is not clear if you are describing the current state of things, or are proposing a new set of IR forms. If the latter, it seems like there should be a part of the document that describes how it is different from the current state, and describe WHY it is better.